lib/pf/file_paths.pm

```diff
@@ -81,13 +82,15 @@ our (
     $radius_filters_config_file,
     $billing_tiers_config_file,
     $dhcp_filters_config_file,
-    $dns_filters_config_file,
+    $dns_filters_config_file, $dns_filters_default_config_file,
```

Comment:
This is not consistent, some new default file variables are on a new line, some on the same line as the non-default one.

Please make it consistent, I think this style (default on same line as the other) looks good